packages/hoppscotch-cli/package.json

```diff
@@ -59,5 +59,8 @@
     "ts-jest": "^27.1.4",
     "tsup": "^5.12.7",
     "typescript": "^4.6.4"
+  },
+  "dependencies": {
+    "zod": "^3.22.2"
```

Comment:
Shouldn't we add this under `devDependencies` similar to other packages such that it gets inlined?